app-creator/code-editor/deserializer.js

```diff
@@ -2,32 +2,235 @@ var mapStyles = require('users/msibrahim/app-creator:map-styles');
 var sidemenu = require('users/msibrahim/app-creator:sidemenu');
 
 exports.createApp = createApp;
+exports.createMultiDeviceApp = createMultiDeviceApp;
+exports.createResponsiveApp = createResponsiveApp;
+exports.createMultiSelectorApp = createMultiSelectorApp;
+
+var MOBILE_WINDOW_SIZE = 900;
+
+/**
+ * Temporary placeholder to avoid breaking apps already created with this API.
+ */
```

Comment:
Is it too early for backward compatibility?

Reply:
I vote to mark it deprecated by throwing an error instead:
```
createResponsiveApp is deprecated, use createMultiDeviceApp instead.
```
But I'll let Michael have the final word here :)

Reply:
It's a good instinct, but I'd err on the side of removing this method entirely (while there are essentially zero callers). Do you have reason to think that anyone relies on the existence of this method today?

Reply:
Just anyone who started playing around with the app creator after the project presentation.

Reply:
Let's remove it then - reduced code complexity, particularly in the form of a smaller API surface, is well worth breaking some early dogfood users.